.github/scripts/trymerge.py

```diff
@@ -1486,24 +1487,53 @@ def get_rockset_results(head_sha: str, merge_base: str) -> List[Dict[str, Any]]:
         return []
 
 
+REMOVE_JOB_NAME_SUFFIX_REGEX = re.compile(r", [0-9]+, [0-9]+, .+\)")
```

Comment:
NIT: Make sure that this brace is at the end of the statement, and perhaps specify that regex includes the entire suffix 
```suggestion
REMOVE_JOB_NAME_SUFFIX_REGEX = re.compile(r"\(\w+ , [0-9]+, [0-9]+, .+\)$")
```